release/models/acl/openconfig-acl.yang

```diff
@@ -830,6 +838,61 @@ module openconfig-acl {
     }
   }
 
+  grouping prefix-list-config {
+     leaf name {
+        type string;
+        description
+           "Name of the prefix list to
+            be used in the matches";
+      }
+      leaf description {
+        type string;
+        description
+          "Prefix list description";
+      }
+      leaf-list prefix {
```

Comment:
This leaves open the possibility of a mixed mode of prefix-lists, IPv4 and IPv6. Do we want that to be the case (generally, we've discussed that this isn't widely supported)? If not, does a `prefix-list` itself need a mode?

Reply:
Agree to have separated IPv4 and IPv6 prefix lists. It is supported in most platforms.